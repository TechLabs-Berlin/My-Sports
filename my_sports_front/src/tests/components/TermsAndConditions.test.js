import React from "react";
import { shallow } from "enzyme";
import TermsAndConditions from "../../components/TermsAndConditions";
import "../setupTests";

test("should render terms and conditions correctly", () => {
  const wrapper = shallow(<TermsAndConditions />);
  expect(wrapper.debug()).toMatchSnapshot();
});
