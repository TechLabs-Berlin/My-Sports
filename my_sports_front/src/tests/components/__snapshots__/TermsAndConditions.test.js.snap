// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render terms and conditions correctly 1`] = `
"<div>
  <h2 className=\\"terms-header\\">
    Terms &amp; Conditions
  </h2>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae, dolores, quis similique obcaecati sequi id quo iste maxime velit consectetur praesentium consequatur labore doloribus voluptates animi laboriosam facilis. Quaerat, ipsum. Quisquam incidunt blanditiis, excepturi ullam architecto beatae consectetur rem nisi consequatur laboriosam, facilis temporibus fugit sunt veritatis alias voluptatibus! Corrupti aliquid a doloremque dolore, sunt sit ipsum laudantium nobis. Neque. Nihil quo ex, expedita at deleniti necessitatibus earum eos rerum? Vel deserunt accusamus veniam dolorum, minus nulla esse illum similique eius ab modi! Placeat voluptas nam debitis consequuntur, voluptate eum. Error cumque eveniet sequi quibusdam inventore eum minus quis, asperiores alias, reiciendis hic aliquid amet impedit adipisci similique soluta ex obcaecati ea ullam maxime vero beatae iure? Mollitia, tempore. Nihil. In reprehenderit fuga qui doloribus voluptate. Ratione, autem. Sit sunt libero beatae voluptates eligendi accusantium magni delectus officiis molestias facere voluptatum, nisi excepturi voluptatibus neque velit adipisci cumque in totam. Repudiandae nemo quaerat praesentium maxime dolorum enim asperiores similique earum illum ab, tempora vero, ad, rerum in porro quae eum corrupti sed alias temporibus perferendis commodi aperiam consequatur! Consectetur, cupiditate! Recusandae autem obcaecati quisquam officiis maxime dolorum voluptas molestias. Velit, ipsum inventore? Totam facilis, hic tempore commodi ea natus fuga incidunt odio doloribus quos facere illo. Quis, doloremque! Ipsa, inventore! Dolorem deleniti itaque odio, repudiandae ducimus provident quod voluptas voluptatibus vel sit perferendis fugit labore. Consectetur minus ut quidem, incidunt fugiat quibusdam recusandae animi neque ipsam culpa odio, eius optio? Nisi tenetur id fugiat sapiente suscipit impedit, beatae dolore natus saepe quidem accusantium ab fuga libero velit facere sint? Nesciunt earum maiores molestiae illo harum. Ab mollitia id placeat nostrum. Aliquid quod debitis qui earum pariatur commodi ea, veniam molestias, veritatis explicabo eveniet, alias nemo itaque assumenda dolorum eum magni vero quam minus dicta. Non magnam minus natus possimus nihil.
  </p>
</div>"
`;
